lib/subscriptions_test_kit/common/subscription_conformance_verification.rb

```diff
@@ -77,8 +77,8 @@ def server_check_channel(subscription, access_token)
           Added the Authorization header field with a Bearer token set to #{access_token} to the `header` field on the
           Subscription resource in order to connect successfully with the Inferno subscription channel.
         ))
-        subscription['header'] = [] unless subscription['header'].present?
-        subscription['header'].append("Authorization: Bearer #{access_token}")
+        subscription_channel['header'] = [] unless subscription['header'].present?
```

Comment:
Seems like you'd need this change as well?

```suggestion
        subscription_channel['header'] = [] unless subscription_channel['header'].present?
```